### PR TITLE
feat: Add fail2ban_exporter role

### PR DESCRIPTION
**Pull request by anviar**
Add new role: **fail2ban_exporter**

### CODE REVIEW

File: roles/fail2ban_exporter/vars/main.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+go_arch_map:
+  i386: '386'
+  x86_64: 'amd64'
+  aarch64: 'arm64'
+  armv7l: 'armv7'
+  armv6l: 'armv6'
+
+go_arch: "{{ go_arch_map[ansible_architecture] | default(ansible_architecture) }}"
+_gitlab_projectid: 24199687
```
Comment (SuperQ) [inline roles/fail2ban_exporter/vars/main.yml:10]: I guess we need to update `.github/scripts/version_updater.sh` to handle checking for new versions from GitLab projects.

**Review by SuperQ — APPROVED** [review]
Not going to block this on having auto-updates. But it would be good to followup with some refactoring to handle github vs gitlab.

**Comment by anviar** [comment]
This project have github mirror, but I prefer to use original source.
I will check `.github/scripts/version_updater.sh`, may be I can help with that.
Should I open another PR for that update or use this one?

**Comment by SuperQ** [comment]
A new PR is fine.

**Pull request merged**